spacemacs/packages.el

```diff
@@ -1987,7 +1987,13 @@ Put (global-hungry-delete-mode) in dotspacemacs/config to enable by default."
 (defun spacemacs/init-org-bullets ()
   (use-package org-bullets
     :defer t
-    :init (add-hook 'org-mode-hook 'org-bullets-mode)))
+    :init (add-hook 'org-mode-hook 'org-bullets-mode)
+    :config (setq org-bullets-bullet-list '("■" "◆" "▲" "▶"))))
```

Comment:
you should put it in `:init` to allow users to easily override the setting in their config.